repo/packages/S/scale/0/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "scm": "https://github.com/ngageoint/scale.git",
+  "postInstallNotes": "The Scale DCOS Service has been successfully installed!\n\n\tDocumentation: https://ngageoint.github.io/scale/\n\tIssues: https://github.com/ngageoint/scale/issues",
+  "maintainer": "http://www.nga.mil",
```

Comment:
change this to an email id where users can reach out for questions related to the package.